template/user.email.update/body.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Updated Email Address</title>
+  <style>
+    tr td ul li img {
+      max-width: 500px;
+    }
+
+    tr td pre {
+      overflow: auto;
+    }
+
+    a:hover {
+      text-decoration: underline !important;
+    }
+  </style>
+</head>
+
+<body style="
+        margin: 10px;
+        padding: 0;
+        background-color: #f9f9f9;
+        font-family: 'Open Sans', sans-serif;
+        font-size: 14px;
+        font-weight: lighter;
+        line-height: 1.2;">
+  <div style="
+        min-height: 300px;
+        width: 100%;
+        max-width: 750px;
+        margin: 0 auto;
+        padding: 20px;
+        border: 1px solid #d7d7d7;
+        border-radius: 4px;
+        background-color: #fff;
+        box-shadow: 0 2px 4px #d7d7d7;">
+    <div style="
+          width: 100%;">
+      <img src="{{.webURL}}/_openshiftio/assets/images/openshift_logo_black.png" width="25%" height="45px" />
+      <div style="
+            width: 95%;
+            margin: 10px auto;
+            border: 1px solid #d7d7d7;">
+      </div>
+    </div>
+    <div style="
+          background-color: #f9f9f9;
+          padding: 10px 20px;
+          min-height: 150px;
+          max-width: 700px;
+          text-align: left;">
+      <h3 style="margin-bottom: 5px;">
+        <a href="{{.custom.VerifyURL }}" style="
```

Comment:
Normal would be a camelcase style variable..

Reply:
https://github.com/fabric8-services/fabric8-notification/pull/51/commits/071502685301a39eee6a9f2edd45e5466b720dbb